deployment/ccip/changeset/testhelpers/test_helpers_solana_v0_1_0.go

```diff
@@ -462,6 +463,49 @@ func CCIPSendCalldata(
 	return calldata, nil
 }
 
+// GenMessagesForMulticall3 generates the calls and total value for the messages so that they can be used
+// with a multicall3 transaction.
+// Note that this is EVM-specific.
+func GenMessagesForMulticall3(
```

Comment:
You should probably use the `v0_1_1` package.... not sure what the long term plan is for these things.